addons/binding/org.openhab.binding.coolmasternet/README.md

```diff
@@ -23,43 +22,18 @@ Bridge coolmasternet:controller:main [ host="192.168.0.100" ] {
 
 ## Channels
 
-<table>
-	<tr><td><b>Channel</b></td><td><b>Item Type</b></td><td><b>Description</b></td></tr>
-	<tr>
-		<td>on</td>
-		<td>Switch</td>
-		<td>Turn HVAC unit on and off.</td>
-	</tr>
-	<tr>
-		<td>mode</td>
-		<td>String</td>
-		<td>HVAC Mode. "Auto", "Cool", "Heat", "Dry" or "Fan Only". Unit may not support all modes.</td>
-	</tr>
-	<tr>
-		<td>fan</td>
-		<td>String</td>
-		<td>Fan Mode. "Auto", "Low", "Medium", "High" or "Top". Unit may not support all speeds.</td>
-	</tr>
-	<tr>
-		<td>set_temp</td>
-		<td>Number</td>
-		<td>Temperature target setpoint.</td>
-	</tr>
-	<tr>
-		<td>current_temp</td>
-		<td>Number</td>
-		<td>Current temperature at HVAC unit.</td>
-	</tr>
-	<tr>
-		<td>louvre</td>
-		<td>String</td>
-		<td>Louvre angle. "No Control", "Auto Swing", "Horizontal", "30 degrees", "45 degrees", "60 degrees" or "Vertical". Unit may not support all angles.</td>
-	</tr>
-</table>
+| Channel      | Item Type | Description                                                                                                                                      |
+|--------------|-----------|--------------------------------------------------------------------------------------------------------------------------------------------------|
+| on           | Switch    | Turn HVAC unit on and off.                                                                                                                       |
+| mode         | String    | HVAC Mode. "Auto", "Cool", "Heat", "Dry" or "Fan Only". Unit may not support all modes.                                                          |
+| fan          | String    | Fan Mode. "Auto", "Low", "Medium", "High" or "Top". Unit may not support all speeds.                                                             |
+| set_temp     | Number    | Temperature target setpoint.                                                                                                                     |
+| current_temp | Number    | Current temperature at HVAC unit.                                                                                                                |
+| louvre       | String    | Louvre angle. "No Control", "Auto Swing", "Horizontal", "30 degrees", "45 degrees", "60 degrees" or "Vertical". Unit may not support all angles. |
```

Comment:
THANK YOU 😄

Reply:
I have fixed "direct html tables" wherever possible.
But i didn't check how to handle nested tables in markdown, so i didn't touch those.